applications/useink_light_clients.md

```diff
@@ -0,0 +1,79 @@
+- **Website:** https://vadim.blog/
+
+### Legal Structure
+
+- **Registered Address:** Municipiul Braşov, Strada LUNGĂ, Nr. 149, Ap. P3, Judet Braşov
+- **Registered Legal Entity:** WEB3SOLUTIONS S.R.L.
+
+### Team's experience
+
+I have extensive experience in both back-end and front-end development using Go, Node.js, React, GraphQL, and TypeScript. My projects span across various industries such as e-commerce, event management, and non-profit, and have contributed to the development of the Ink dApp on the Substrate blockchain. I am also well-versed in open-source contributions and have a diverse skill set, making me confident in my ability to deliver impactful solutions to complex problems.
+
+### Team Code Repos
+
+- https://github.com/substrate-dapps/ink-dApp-directory
+- https://github.com/nicolad/useink-kitchen-sink
+
+Please also provide the GitHub accounts of all team members. If they contain no activity, references to projects hosted elsewhere or live are also fine.
+
+- https://github.com/nicolad
+
+### Team LinkedIn Profiles (if available)
+
+- https://www.linkedin.com/in/vadimnicolai/
+
+## Development Status :open_book:
+
+- https://github.com/paritytech/useink/issues/40
+
+## Development Roadmap :nut_and_bolt:
+
+I will implement features based on the issue requirements and will provide detailed documentation on the expected functionality of the issue. I will also ensure that all deliverables are thoroughly tested and verified before submission.
+
+### Overview
+
+- **Total Estimated Duration:** 2 weeks
+- **Full-Time Equivalent (FTE):** 1 FTE
+- **Total Costs:** 5,000 USD
+
+### Milestone 1
+
+- **Estimated duration:** 2 weeks
+- **FTE:** 1
+- **Costs:** 5,000 USD
+
+| Number | Deliverable               | Specification                                                                                                                                                           |
+| ------ | ------------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| 0a.    | License                   | Apache 2.0                                                                                                                                                              |
+| 0b.    | Documentation             | I will provide both inline documentation of the code and a basic tutorial that explains how to use light clients to connect to a Substrate node.                        |
+| 0c.    | Testing and Testing Guide | Core functions will be fully covered by comprehensive unit or smoke tests to ensure functionality and robustness. In the guide, I will describe how to run these tests. |
+| 1.     | Light clients as default  | I will implement light clients as the default method of connecting to a Substrate node, with the option to use third party RPC nodes.                                   |
```

Comment:
Please change this. Light clients should not be the default because they can take a while to initialize. It is a better user experience to use the current implementation. We should let developers or even the user using the dApp to decide which they want to use